package.json

```diff
@@ -9,7 +9,7 @@
     "type": "git",
     "url": "git+https://github.com/teslaworksumn/proton-sequencer.git"
   },
-  "author": "Ryan Fredlund, Byron Ambright, Joshua Guldberg, Alex Kafer, Kelly Larson, Vladimir Kuksenko",
+  "author": "Ryan Fredlund, Byron Ambright, Joshua Guldberg, Alex Kafer, Kelly Larson, Vladimir Kuksenko, Kayla Engelstad",
```

Comment:
Manually added @kayla-e774 so we don't need to deal with the future merging issue.